html/mycakeapp/src/Controller/AuctionController.php

```diff
@@ -223,4 +249,68 @@ public function home2()
 		])->toArray();
 		$this->set(compact('biditems'));
 	}
+
+	//商品の発送に関するページ
+	public function sending($bidinfo_id)
+	{
+		// $idのBidinfoを取得
+		$bidinfo = $this->Bidinfo->get($bidinfo_id, [
+			'contain' => ['Users', 'Biditems', 'Biditems.Users', 'Bidsendings']
+		]);
+		$this->set(compact('bidinfo'));
+		//出品者または落札者でない時はindexページに戻るようアクセス制限をかける
+		//ログインユーザーのID
+		$login_user_id = $this->Auth->user('id');
+		//受け取るユーザー(落札者)のユーザーID
+		$receive_user_id = $bidinfo->user_id;
+		//出品者のユーザーID
+		$sent_user_id = $bidinfo->biditem->user_id;
+		$this->set(compact('login_user_id', 'receive_user_id', 'sent_user_id'));
+
+		if (!($login_user_id === $receive_user_id) && !($login_user_id === $sent_user_id)) {
```

Comment:
(修正不要)
```suggestion
		if (($login_user_id !== $receive_user_id) && ($login_user_id !== $sent_user_id)) {
```
直すほどではないですがこっちの方が分かりやすいですね！個人的には！

Reply:
ユズノハさんの書き方の方が直感的に分かりやすいと感じたので修正不要とありますが修正しました。